src/app/pages/my-virtrolio/my-virtrolio.component.css

```diff
@@ -65,14 +65,43 @@
   overflow-x: auto;
 }
 
-.sharing-link-body > label {
+.sharing-link-body > label, #sharing-link-buttons > label, #sharing-link-buttons > div > label {
   display: none;
 }
 
+#sharing-link-buttons > div {
+  display: inline-block;
+}
+
 #sharing-link-field {
   background: white;
 }
 
+#share-link-social {
+  z-index: 2000;
+}
+
```

Comment:
We aren't using this selector anymore - idk if you need to apply the z-index elsewhere though
```suggestion
```